services/ui-src/src/components/Banner/Banner.tsx

```diff
@@ -1,15 +1,27 @@
 import { Alert } from "@cmsgov/design-system";
 import { BannerData } from "types";
 import { parseLabelToHTML } from "utils";
+import DOMPurify from "dompurify";
+
+const domPurifyBannerConfig = {
+  // Only these tags will be allowed through
+  ALLOWED_TAGS: ["ul", "ol", "li", "a", "#text", "strong", "b", "em"],
+  // On those tags, only these attributes are allowed
+  ALLOWED_ATTR: ["href", "alt"],
+  // If a tag is removed, so will all its child elements & text
+  KEEP_CONTENT: false,
+};
 
 export const Banner = ({ bannerData, ...props }: Props) => {
   return (
     <>
       {bannerData && (
         <Alert heading={bannerData.title} {...props}>
-          <p className="ds-c-alert__text">
-            {parseLabelToHTML(bannerData.description)}
-          </p>
+          <span className="ds-c-alert__text">
```

Comment:
A `<ul>` is not allowed in a `<span>`, I'd suggest using a `<div>`.

From an [HTML validator](https://validator.w3.org/):

<img width="1996" height="424" alt="QMR UL in Span error" src="https://github.com/user-attachments/assets/9d9a7cdb-ed3c-4243-8522-09eced7c68f1" />

Because a `<span>` expects to have [phrasing content](https://html.spec.whatwg.org/multipage/dom.html#phrasing-content-2) as children.

```suggestion
          <div className="ds-c-alert__text">
```